operatorcert/entrypoints/verify_changed_dirs.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+import logging
+import sys
+
+from operatorcert import (
+    get_files_changed_in_pr,
+    verify_changed_files_location,
+    get_repo_and_org_from_github_url,
+)
+
+
+def main() -> None:
+    logging.basicConfig(stream=sys.stdout, level="INFO", format="%(message)s")
+
+    parser = argparse.ArgumentParser(
+        description="Determines the OCP version under test."
+    )
+    parser.add_argument("--operator_name", help="Unique name of the operator package")
+    parser.add_argument(
+        "--operator_version", help="Operator version in the semver format"
```

Comment:
Shouldn't this be the bundle version instead?

Reply:
done